framework/doc/content/source/timesteppers/CompositionDT.md

```diff
@@ -0,0 +1,17 @@
+# CompositionDT
+
+!syntax description /Executioner/TimeStepper/CompositionDT
+
+## Description
+
+The `CompositionDT` TimeStepper takes TimeSteppers as inputs and generates a composed time step size based on the user specific composition rules. A base TimeStepper is required to provide a baseline time step size. The available composition rules including max, min and hit. The max/min option is specified with [!param](/Executioner/TimeStepper/CompositionDT/composition_type). It produces the maximum/minimum time step size value of all the input TimeSteppers. The hit option ensures the TimeStepper hits user specified time. It requires a TimeSequenceStepper with [!param](/Executioner/TimeStepper/CompositionDT/times_to_hit_timestepper) to specify the time sequence to hit. All together, The `CompositionDT` TimeStepper allows users to have a base time step with upper or lower limits and hit specified time points.
+
```

Comment:
since you are doing only one operation per CompositionDT, let s mention that we can nest CompositionDT

Reply:
maybe we shouldnt compose compositionDT

I think it s more simple to just have all the parameters in one?